app/lib/admin/backend.dart

```diff
@@ -247,4 +253,53 @@ class AdminBackend {
       await tx.commit();
     });
   }
+
+  /// Handles PUT '/api/admin/packages/<package>/flutter-favorite'
+  ///
+  /// Note, this API end-point is intentioanlly locked down even if it doesn't
+  /// return anything secret. This is because the /admin/ section is only
+  /// intended to be exposed to administrators. Users can read the status
+  /// through API that returns list of package tags.
+  Future<api.FlutterFavoriteStatus> handleGetFlutterFavorite(
+    String packageName,
+  ) async {
+    await _requireAdminPermission(AdminPermission.manageFlutterFavorite);
+    final package = await packageBackend.lookupPackage(packageName);
+    if (package == null) {
+      throw NotFoundException.resource(packageName);
+    }
+
+    return api.FlutterFavoriteStatus(
+      isFlutterFavorite: package.isFlutterFavorite,
```

Comment:
I will assume that we are not concerned about the existing entities returning null (before the backfill happens).